doc/reference/cardano-node-cli-reference.md

```diff
@@ -98,5 +99,45 @@ The `genesis` command contains the following sub commands:
 * `hash`: retrieves the hash value
 
 *cardano-cli text-view*
-The `text-view` command contains the following sub command:
-* `decode-cbor`: prints a text view file, as decoded CBOR.
+The `text-view` command contains the following subcommand:
+* `decode-cbor`: prints a text view file as decoded CBOR.
+
+
+
+## Advanced Commands
+
+`kes-period-info`: This command runs the following checks on your operational certificate and your operational certificate issue counter:
```

Comment:
This makes it look like `kes-period-info` is a top-level command, where as it is only a subcommand of `query`.